crates/router/src/connector/iatapay/transformers.rs

```diff
@@ -58,6 +59,12 @@ pub struct RedirectUrls {
     failure_url: String,
 }
 
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct PayerInfo {
+    token_id: Secret<String>,
```

Comment:
What is this token for, add a one line comment

Reply:
Hey, currently this token is used to pass vpa_id (as suggested by Iatapay), but for different payment method it will be different payment_method_data. So, it is just a format of request for Iatapay, we can't define any fix thing for it in the code.